buildpipeline/DotNet-CoreClr-Trusted-Mac.json

```diff
@@ -12,7 +12,7 @@
         "definitionType": "task"
       },
       "inputs": {
-        "filename": "$(Agent.BuildDirectory)/s/clean.sh",
+        "filename": "$(Agent.BuildDirectory)/s/clean.sh -all",
```

Comment:
The `-all` needs to be an argument, not the file name. This is interpreted as a file with space in it, "`clean.sh -all`".

Build break: https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_build?_a=summary&buildId=1131260